packages/mobile/src/account/BankAccounts.test.tsx

```diff
@@ -28,13 +28,13 @@ const MOCK_BANK_ACCOUNTS = {
 
 const mockGetResponse = Promise.resolve({
   ok: true,
-  json: () => MOCK_BANK_ACCOUNTS,
+  json: () => Promise.resolve(MOCK_BANK_ACCOUNTS),
 })
 const mockDeleteResponse = Promise.resolve({
   ok: true,
 })
 jest.mock('src/in-house-liquidity', () => ({
-  getFinclusiveBankAccount: jest.fn(() => mockGetResponse),
+  getFinclusiveBankAccounts: jest.fn(() => mockGetResponse),
```

Comment:
code golf, I don't actually care much though
```suggestion
  getFinclusiveBankAccounts: jest.fn().mockReturnValue({ok: true, json: jest.fn().mockResolvedValue(MOCK_BANK_ACCOUNTS)),
```